src/aat/resources/features/F-049/S-234.td.json

```diff
@@ -0,0 +1,46 @@
+{
+  "_guid_": "S-234",
+  "_extends_": "F-049_Test_Data_Base",
+  "title": "must return 400 when request provide Special character for Case type ID in CCD",
+  "specs": [
+    "a detailed profile in CCD",
+    "Start case creation as Case worker",
+    "CCD Data Store",
+    "provide Special character for Case type ID",
+    "code is HTTP-400"
+  ],
+  "request": {
+    "pathVariables": {
+      "ctid": "@@@@@"
+    }
+  },
+  "expectedResponse": {
+    "responseCode" : 500,
+    "responseMessage" : "Internal Server Error",
+    "headers" : {
+      "Cache-Control" : "no-cache, no-store, max-age=0, must-revalidate",
+      "Connection" : "close",
+      "Content-Encoding" : "gzip",
+      "Content-Type" : "application/json;charset=UTF-8",
+      "Date" : "[[ANY]]",
+      "Expires" : "0",
+      "Pragma" : "no-cache",
+      "Transfer-Encoding" : "chunked",
+      "vary" : "accept-encoding",
+      "X-Content-Type-Options" : "nosniff",
+      "X-Frame-Options" : "DENY",
+      "X-XSS-Protection" : "1; mode=block"
```

Comment:
these headers and the body below could be inherited from a common response object